Sources/SwiftDocC/Infrastructure/CoverageDataEntry.swift

```diff
@@ -104,7 +104,6 @@ public struct CoverageDataEntry: CustomStringConvertible, Codable {
     }
 
     /// Outputs a short table summarizing the coverage statistics for a list of data entries.
-    /// - Parameter coverageInfo: An array of entries to summarize.
```

Comment:
Why remove this?

Reply:
Because it's an experimental API that result in warnings about not having all of its parameters documented.